bodo/pandas/utils.py

```diff
@@ -537,6 +537,9 @@ def df_to_cpp_table(df):
     if any(col.num_chunks == 0 for col in arrow_table.columns):
         arrow_table = pa.Table.from_arrays(new_columns, schema=arrow_table.schema)
 
+    if return_schema:
```

Comment:
Isn't the normal pattern here to always return schema and then do "ret, _" in cases wehre you don't need it, particularly for cases like this where the function isn't used very many times?